MediaBrowser.MediaEncoding/Probing/FFProbeHelpers.cs

```diff
@@ -93,9 +93,11 @@ public static string GetDictionaryValue(IReadOnlyDictionary<string, string> tags
                 return null;
             }
 
-            if (DateTime.TryParse(val, DateTimeFormatInfo.CurrentInfo, DateTimeStyles.AssumeUniversal, out var i))
+            DateTime date;
+            if (DateTime.TryParse(val, DateTimeFormatInfo.CurrentInfo, DateTimeStyles.AssumeUniversal, out date) ||
+                DateTime.TryParseExact(val, "yyyy", DateTimeFormatInfo.CurrentInfo, DateTimeStyles.AssumeUniversal, out date))
```

Comment:
Looking at the ffmpeg output, I don't see why this is needed?

Reply:
`DateTime.TryParse` doesn't work on dates that have a simple year, from what I've seen.

Since dates can either be yyyy-mm-dd or yyyy, there are cases where TryParse will return null, hence the added TryParseExact.

It's based on this: https://stackoverflow.com/questions/38948394/datetime-tryparse-fails-at-yyyy-only

If the behavior changed since then though, I can certainly take it out :yum:

Reply:
In that case you should add some tests with files that only have a year